config/schedule.rb

```diff
@@ -20,3 +20,8 @@
   set :output, nil
   rake 'preservation_errors', mailto: 'sdr2service@sul-sdr-services.stanford.edu'
 end
+
+every 1.day, at: '5:30am' do
```

Comment:
I would say you could go with 5:10 or 5:15 if you wanted, but leaving it as 5:30 is unlikely to cause problems.  (Just thinking the timestamps of the emails might be closer)